### DISCUSSION TITLE
The button to turn on the sound or play a video

### DISCUSSION THREAD

**Discussion by dodsik** [AR-js-org/AR.js#546] [Help]
Hello. I use this code of yours:
https://github.com/AR-js-org/AR.js/blob/master/aframe/examples/image-tracking/nft-video/index.html
Video - autoplay, because - muted.
Is it possible to somehow output the audio or video playback button on top of <a-scene>?
I'm trying to do it, but it disappears after <arjs-loader>...

Interested in exactly how to make the button not disappear?
Thanks!

**Comment by jabbourdan** [comment]
Hey @dodsik did you find any thing regarding this ?
